### ISSUE TITLE
Causes severe typing lag after Obsidian v1.7.0

### ISSUE THREAD

**Issue by lethefrost** [liamcain/obsidian-lapel#27]
It's weird, and this problem doesn't seem to be supposed to be caused by this simple plug-in which just displays heading levels, but after troubleshooting, typing resumes smoothness with lapel turned off, and when lapel is turned on again, it causes lag.

**Comment by garethfoote** [comment]
Same experience for me on 1.7.7. Hadn't noticed if this was the case for previous versions or not.

**Comment by warm-july** [comment]
yes, I'm having the same issue, what could possibly be the cause
